contingency/contingency-api/src/test/java/com/powsybl/contingency/ContingencyTest.java

```diff
@@ -29,16 +28,19 @@
 
     @Test
     public void test() {
-        ContingencyElement element1 = new BranchContingency("line");
-        ContingencyElement element2 = new GeneratorContingency("generator");
-        Contingency contingency = new Contingency("contingency", element1, element2);
+        Contingency contingency = Contingency.builder("contingency")
+                .branch("line")
```

Comment:
Maybe we should add a prefix like 'add' to better figure out that we add somthing to the builder and don't replace (like in guava ImmutableList) ?
```java
Contingency contingency = Contingency.builder("contingency")
    .addBranch("")
    .addGenerator("")
   .build()            
```